test/docker/networktest/DockerRemoting.Tests.ps1

```diff
@@ -0,0 +1,60 @@
+$imageName = "remotetestimage"
+Describe "Basic remoting test with docker" -tags @("Scenario","Slow"){
+    BeforeAll {
+        $dockerimage = docker images --format "{{ .Repository }}" $imageName
+        if ( $dockerimage -ne $imageName ) {
+            $pending = $true
+            write-warning "Docker image '$imageName' not found, not running tests"
+            return
+        }
+        else {
+            $pending = $false
+        }
+        # this is all I could figure out how to set up a container which
+        # sticks around for a while
```

Comment:
What about the container lifecycle commands? `docker container start`, `docker exec`, `docker container stop`, etc? `docker ps` and `docker container list` both show running containers.

Using those commands should prevent the need to use start-sleep.